test/e2e/framework/deployment/logging.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
2017 - you probably made a new file here but basing it off what is seen by git

Reply:
Should I keep the 2017 for all files then?

Reply:
2019 only if it's a new file. If you're moving a file, keep the old copy right date

Reply:
Changed it to "2017" for all files that have code straight from deployment_util.go.

Reply:
fixed this

Reply:
Thanks, general rule (that I follow at least) is new files according to git should have the current year. Any file "renamed" should keep whatever date is already there